modules/core/LocationManager.js

```diff
@@ -263,7 +268,7 @@ export class LocationManager extends EventEmitter {
    * Returns a locationSetID for a given locationSet (fallback to `+[Q2]`, world)
    * (The locationSet doesn't necessarily need to be resolved to compute its `id`)
    *
-   * @param  `locationSet`  A locationSet Object, e.g. `{ include: ['us'] }`
+   * @param  `locationSet`  A {LocationSet} Object, e.g. `{ include: ['us'] }`
```

Comment:
Here's the original doc text just with the new LocationSet type mentioned. Did I get the shape of the object correct? Here's how it looks in the docs: 


![image](https://user-images.githubusercontent.com/1887955/216102406-839958c4-ac56-4f12-ad61-0e79c0302891.png)

![image](https://user-images.githubusercontent.com/1887955/216102686-187f565a-442c-4199-8023-a9dcab4aa94c.png)